components/README.md

```diff
@@ -16,7 +16,8 @@ This table lists the available types:
 | type|description|example|
 |--|--|--|
 |controller|A [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/) which reacts on a standard Kubernetes resource or manages [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) resources. The component's name reflects the name of the primary resource it controls.|namespace-controller|
-|operator|is a [Kubernetes Operator](https://coreos.com/operators/) covering  application specific logic for operatiion of the application, like steps to upscale a stateful applicatin. It usually reacts on changes to resources of a [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. |application-operator|
+|controller-manager|A daemon that embeds all [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/)s of a domain. Such approach can bring benefits in operations in contrast to shipping all controllers individual. A `controller-managaer` is named by the domain it s belonging to. |asset-store-controller-manager|
+|operator|is a [Kubernetes Operator](https://coreos.com/operators/) covering application specific logic for operatiion of the application, like steps to upscale a stateful applicatin. It usually reacts on changes to resources of a [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. |application-operator|
```

Comment:
```suggestion
|operator|is a [Kubernetes Operator](https://coreos.com/operators/) which covers the application-specific logic behind the operation of the application, such as steps to upscale a stateful application. It reacts on changes made to custom resources derived from a given [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. |application-operator|
```
I assume the custom resources are based on a given CustomResourceDefinition - I wanted to point this out.